.github/workflows/ci.yml

```diff
@@ -32,20 +32,11 @@ jobs:
       fail-fast: false
       matrix:
         ruby-version:
-          - 2.4
-          - 2.5
           - 2.6
           - 2.7
         gemfile:
-          - rails4.2
```

Comment:
Shall we update https://github.com/zendesk/active_record_host_pool/blob/master/active_record_host_pool.gemspec#L27 
```
s.add_runtime_dependency("activerecord", ">= 4.2.0", "< 6.1")
```

to be 
```
s.add_runtime_dependency("activerecord", ">= 5.2.0", "< 6.1")
```
?

Reply:
Yep 👍